src/js-marketing-notification.ts

```diff
@@ -8,15 +9,15 @@ export interface Notification {
 export class JsMarketingNotification {
   private notificationContainer: Element;
   
-  constructor(private popupTitle: string) {}
+  constructor(private notificationTitle: string) {}
 
   public output(notification: Notification): void {
     this.notificationContainer = document.createElement('div');
     this.notificationContainer.classList.add('marketing-notification');
     if (this.isFirstPageVisit())
       this.notificationContainer.classList.add('expanded');
 
-    const tab = this.createNotificationTab(this.notificationContainer);
+    const tab = this.createNotificationTab(this.notificationContainer, notification.tabContent);
     const body = this.createNotificationBody(notification);
```

Comment:
I know this is not a change for this PR, but you can do these as comma separated variable declarations.